src/stylesheets/core/_utilities.scss

```diff
@@ -207,10 +207,11 @@
     }
 
     &.usa-current { /* stylelint-disable-line selector-no-qualifying-type */
-      border-left: 0.4rem solid $color-primary;
+      $current-link-border: 0.4rem;
```

Comment:
I don't know if we need to add this at this variable at this point, but maybe could think about it in the future. I'd want to have have a bit of research to back up what people are needing from a variable for this border and then determine where it should live (in `variable.scss` etc...) and documentation for it. The name `$current-link-border` implies the entire border value (`0.4rem solid $color-primary;`). If it were just the width, I'd expect the name `$current-link-border-width`.

Reply:
I wasn't thinking of this as a user-modifiable var, just a placekeeping var to keep our math consistent on line 214. But maybe this is a useful var to expose. We could include it in the side navigation docs (since the primary nav seems not to have its own docs).

What about `$current-highlight-border-width` as a var name?